rust/codegen/Cargo.toml

```
@@ -3,9 +3,9 @@ name = "suborbital-macro"
 version = "0.15.1"
```

```suggestion
version = "0.16.0"
```

Should this be bumped now?

---

I would hold off until we're ready to release.